src/components/screens/App.tsx

```diff
@@ -18,6 +23,10 @@ export default function App() {
       void navigate({ to: "/connect", search: (search) => search });
     }
   }, [hasSessions, navigate]);
+  
+  useEffect(() => {
+     analytics.start(userId, walletInfo);
+  }, [analytics, userId, walletInfo])
```

Comment:
Hey that's not really what I had in mind here, I think we can remove the start and do the logic in the useAnalytics directly or do that in an atom